### ISSUE TITLE
Need modifier name for not touching the pad

### ISSUE THREAD

**Issue by kozec** [kozec/sc-controller#291]
After adding `pressed` and `released` modifiers (#290), I'd like to add `touched` that will activate when one puts finger on pad. I have problem with finding name of reversed action - I cannot use `released`, as pad is also button, so it wouldn't be clear to user nor program.

Any ideas are welcomed.

**Comment by LinuxJoeUser** [comment]
I think the adjective "untouched" is my first choice because it *looks* like the opposite of "touched."  And, although it implies a state of disuse, of being ["not handled, used, or tasted."](https://en.oxforddictionaries.com/definition/untouched) more than it means released / let-go, it's meaning is close enough.

I've looked through [Roget's Thesaurus](http://www.bartleby.com/110/), and there are several words that would *technically* work (disengage, loose, unhand, free),  though I feel they're all a bit too old-fashioned in their use as a synonym for "release."  

P.S.  I did find a 15th century word, ["manumit,"](https://en.oxforddictionaries.com/definition/manumit) which comes from a Latin word that literally means ["to send from one's hand."](https://www.etymonline.com/word/manumit)  Unfortunately, it was only used regarding freeing slaves.  That, and no one living knows what it means anymore. :P
